github/resource_github_user_invitation_accepter_test.go

```diff
@@ -0,0 +1,74 @@
+package github
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccGithubUserInvitationAccepter_basic(t *testing.T) {
+	resourceName := "github_repository_collaborator.test"
+	repoName := fmt.Sprintf("tf-acc-test-collab-%s", acctest.RandString(5))
+
+	inviteeToken := os.Getenv("GITHUB_TEST_COLLABORATOR_TOKEN")
```

Comment:
Nit: Should we name this `GITHUB_ALTERNATE_ACCOUNT_TOKEN` or something so its reusable in other contexts if necessary?

Reply:
Well, the token is always associated with a user and what that user has (or doesn't have) access to, so I feel that the ENV variable name should hint that the token belongs to `GITHUB_TEST_COLLABORATOR`, which we (maybe confusingly?) expect to _not_ to be a member of the org, so we can test collaborator invitations.